src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -33,11 +94,11 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
             try
             {
                 // Give application opportunity to find from a different location, adjust, or reject token
-                var messageReceivedContext = new MessageReceivedContext(Context, Options);
+                var messageReceivedContext = new MessageReceivedContext(Context, Scheme, Options);
 
                 // event can set the token
-                await Options.Events.MessageReceived(messageReceivedContext);
-                if (messageReceivedContext.CheckEventResult(out result))
+                await Events.MessageReceived(messageReceivedContext);
+                if (messageReceivedContext.IsProcessingComplete(out result))
```

Comment:
Yeah this is internal plumbing right?  This typically isn't called by event handlers is it?